cypress/support/commands.js

```diff
@@ -1 +1,97 @@
+import AWS from 'aws-sdk';
 
+AWS.config = new AWS.Config({
+  accessKeyId: Cypress.env('AWS_ACCESS_KEY_ID'),
+  secretAccessKey: Cypress.env('AWS_SECRET_ACCESS_KEY'),
+  sessionToken: Cypress.env('AWS_SESSION_TOKEN'),
+  region: 'eu-west-1',
+});
+
+const identityServiceProvider = new AWS.CognitoIdentityServiceProvider();
+
+const USER_POOL_ID = 'eu-west-1_Fto5AuFGa';
+const CLIENT_ID = '55h2kkdaljjq693fsp1h05tbbj';
+
+const TEMP_PASSWORD = 'Abcdefghijkl%9';
+const NEW_PASSWORD = 'NewPassword%0';
+
+const AUTH_FLOW = 'ADMIN_USER_PASSWORD_AUTH';
+const NEW_PASSWORD_REQUIRED = 'NEW_PASSWORD_REQUIRED';
+
+const CUSTOM_IDENTIFIERS = 'custom:identifiers';
+const CUSTOM_ORG_LEGAL_NAME = 'custom:orgLegalName';
+const CUSTOM_ORGN_UMBER = 'custom:orgNumber';
+const CUSTOM_APPLICATION = 'custom:application';
+const CUSTOM_APPLICATION_ROLES = 'custom:applicationRoles';
+const CUSTOM_COMMON_NAME = 'custom:commonName';
+const CUSTOM_FEIDE_ID = 'custom:feideId';
+const CUSTOM_AFFILIATION = 'custom:affiliation';
+
+
+Cypress.Commands.add('addUser', (userName) => {
+
+  const testUser = {
+    UserPoolId: USER_POOL_ID,
+    Username: userName,
+  }
+
+  const createUser = {
+  TemporaryPassword: TEMP_PASSWORD,
+      UserAttributes: [
+        { Name: CUSTOM_IDENTIFIERS, Value: 'feide:test@unit.no'},
+        { Name: CUSTOM_ORG_LEGAL_NAME, Value: 'Unit'},
+        { Name: CUSTOM_ORGN_UMBER, Value: 'NO818477822'},
+        { Name: CUSTOM_APPLICATION, Value: 'NVA'},
+        { Name: CUSTOM_APPLICATION_ROLES, Value: 'Publisher'},
+        { Name: CUSTOM_COMMON_NAME, Value: 'Test User'},
+        { Name: CUSTOM_FEIDE_ID, Value: 'test@unit.no'},
+        { Name: CUSTOM_AFFILIATION, Value: '[member, employee, staff]'},
+      ]
+  }
+
+  identityServiceProvider.adminGetUser(testUser, (err, data) => {
+    if(data) {
+      identityServiceProvider.adminDeleteUser(testUser, () => {});
+    }
+  });
```

Comment:
skjønner ikke helt hva som skjer her. er det bare for å nullstille etter hver test? hva skjer om `err` blir returnert?

Reply:
Nullstilling, hvis err returneres betyr det at brukeren ikke eksisterer. Kan grave litt for å se om det er andre feilsituasjoner.